packages/webpack-extraction-plugin/src/babelPluginStripGriffelRuntime.ts

```diff
@@ -101,16 +114,26 @@ export const babelPluginStripGriffelRuntime = declare<
              */
             CallExpression(path) {
               const calleePath = path.get('callee');
+              const argumentPaths = path.get('arguments');
 
-              if (!calleePath.isIdentifier({ name: '__styles' })) {
-                return;
-              }
+              let argumentPath: typeof argumentPaths[number];
+              let functionKind: FunctionKinds;
 
-              calleePath.replaceWith(t.identifier('__css'));
+              if (calleePath.isIdentifier({ name: '__styles' })) {
+                argumentPath = argumentPaths[1];
+                functionKind = '__styles';
 
-              const argumentPaths = path.get('arguments');
+                calleePath.replaceWith(t.identifier('__css'));
+              } else if (calleePath.isIdentifier({ name: '__resetStyles' })) {
+                argumentPath = argumentPaths[2];
+                functionKind = '__resetStyles';
+
+                calleePath.replaceWith(t.identifier('__resetCSS'));
+              } else {
+                return;
+              }
 
-              if (argumentPaths.length !== 2) {
+              if (functionKind === '__styles' && argumentPaths.length !== 2) {
```

Comment:
nit: you could just parametrize the two errors ?

Reply:
you could actually parametrize a lot:
* argument position
* argument length
* replacement function kind

It avoids some duplication, but it's not  a lot so it's your call

Reply:
Done in 99a99ac